test/libsolidity/semanticTests/externalSource/_relative_imports/contract.sol

```diff
@@ -0,0 +1,26 @@
+import "./dir/a.sol" as a;
+import "./dir/B/b.sol" as b;
+import "./c.sol" as c;
+import "./D/d.sol" as d;
+import "./E/../F/../G/./g.sol" as g;
+import "../../../../_relative_imports/h.sol" as h;
```

Comment:
I'd move `contract.sol` and `G/g.sol` to `dir/` and change the imports to:

```suggestion
import "./a.sol" as a;
import "./B/b.sol" as b;
import "../c.sol" as c;
import "../D/d.sol" as d;
import "./E/../F/../G/./g.sol" as g;
import "../../../../_relative_imports/h.sol" as h;
```

This way you would be able to test the `../`. This is actually how I intended this to work but looks like I made a mistake in my test comment. Sorry for that :)